src/api/callback.rs

```diff
@@ -16,9 +16,7 @@ pub mod callback {
     impl Handle {
         #[napi]
         pub fn disconnect(&mut self) {
-            if let Some(handle) = self.handle.take() {
-                handle.disconnect();
-            }
+            self.handle = None;
```

Comment:
I'm not too familiar with rust but I *think* this is the right way to call `drop()` on the handle?
This was changed in steamworks-rs [here](https://github.com/Noxime/steamworks-rs/pull/241/files#diff-2795162389b3db4685d18117cb326d8790376dc2af8d11d21d3ff84484a15c2aR28-R32).

Reply:
This is now outdated, please see https://github.com/ceifa/steamworks.js/pull/196/files#r2216549985